lib/midcom/helper/imagepopup/style/midcom_helper_imagepopup_upload.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+echo json_encode(array('location' => $data['location']));
```

Comment:
instead of having this style element (which can't really be customized) you could also just return a `midcom_response_json` from the handler function